Octokit/Models/Response/EventInfo.cs

```diff
@@ -204,6 +204,23 @@ public enum EventInfoState
         /// url of the reference's source.
         /// </summary>
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Crossreferenced")]
-        Crossreferenced
+        Crossreferenced,
+
+        /// <summary>
+        /// The issue was reveiewed.
+        /// </summary>
+        Reviewed,
+
+        /// <summary>
+        /// A line comment was made.
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Linecommented")]
+        Linecommented,
```

Comment:
Yeah so on looking into this more I realised we dont even need to tag with the `Parameter()` since these are only response enums and our deserializer removes underscores and does a case insensitive `Enum.Parse()` anyway :+1: